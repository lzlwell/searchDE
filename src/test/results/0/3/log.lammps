LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.26699 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0    414.45302    414.45302      17.1317           10           50 
       1     -118.353     -118.353      17.1317           10           50 
       2   -366.19208   -366.19208      17.1317           10           50 
       3   -522.79632   -522.79632      17.1317           10           50 
       4   -653.29429   -653.29429      17.1317           10           50 
       5   -763.50422   -763.50422      17.1317           10           50 
       6    -878.9846    -878.9846      17.1317           10           50 
       7   -926.63151   -926.63151      17.1317           10           50 
       8   -967.81757   -967.81757      17.1317           10           50 
       9   -1012.3586   -1012.3586      17.1317           10           50 
      10   -1053.6685   -1053.6685      17.1317           10           50 
Loop time of 0.370487 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
         414.453021517     -1012.35857838      -1053.6685404
  Force two-norm initial, final = 7283.03 197.443
  Force max component initial, final = 7198.13 139.575
  Final line search alpha, max atom move = 0.000998174 0.13932
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.151653 (40.9335)
Neigh time (%) = 0.215385 (58.1356)
Comm  time (%) = 0.00141764 (0.382641)
Outpt time (%) = 0.000135183 (0.036488)
Other time (%) = 0.00189614 (0.511797)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2343 ave 2343 max 2343 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46430 ave 46430 max 46430 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46430
Ave neighs/atom = 154.767
Neighbor list builds = 10
Dangerous builds = 10
min_modify line quadratic

clear
