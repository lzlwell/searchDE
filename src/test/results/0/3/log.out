LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.26699 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0    414.45302    414.45302      17.1317           10           50 
       1     -118.353     -118.353      17.1317           10           50 
       2   -366.19208   -366.19208      17.1317           10           50 
       3   -522.79632   -522.79632      17.1317           10           50 
       4   -653.29429   -653.29429      17.1317           10           50 
       5   -763.50422   -763.50422      17.1317           10           50 
       6    -878.9846    -878.9846      17.1317           10           50 
       7   -926.63151   -926.63151      17.1317           10           50 
       8   -967.81757   -967.81757      17.1317           10           50 
       9   -1012.3586   -1012.3586      17.1317           10           50 
      10   -1053.6685   -1053.6685      17.1317           10           50 
Loop time of 0.370487 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
         414.453021517     -1012.35857838      -1053.6685404
  Force two-norm initial, final = 7283.03 197.443
  Force max component initial, final = 7198.13 139.575
  Final line search alpha, max atom move = 0.000998174 0.13932
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.151653 (40.9335)
Neigh time (%) = 0.215385 (58.1356)
Comm  time (%) = 0.00141764 (0.382641)
Outpt time (%) = 0.000135183 (0.036488)
Other time (%) = 0.00189614 (0.511797)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2343 ave 2343 max 2343 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46430 ave 46430 max 46430 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46430
Ave neighs/atom = 154.767
Neighbor list builds = 10
Dangerous builds = 10
